reana.yaml

```diff
@@ -1,17 +1,4 @@
 version: 0.3.0
-code:
-  files:
-   - code/gendata.C
-   - code/fitdata.C
-inputs:
-  parameters:
```

Comment:
Agreed, but I think it has been removed because parameters are being [ignored in current implementation of serial](https://github.com/reanahub/reana-workflow-engine-serial/blob/master/reana_workflow_engine_serial/tasks.py#L86).